### PR TITLE
UI/app state redo

### PR DESCRIPTION
**Pull request by ohmed**
All existing UI should be same and should have same functionality as before.

### CODE REVIEW

File: src/Header.jsx [hunk 1]
```diff
@@ -1,279 +1,365 @@
...
+
+        const handleNonInputKey = ( event ) => {
+
+            switch ( event.which ) {
+
+                case 191: { // /
+
+                    if ( ! state.openedPanel === 'MagicPanel' && ! ioManager.inputFocused() ) {
+
+                        event.preventDefault();
```
[the remark this answers is not in the document]
Comment (avaer) [reply src/Header.jsx:153]: And yes, this is not new code. But since we are rewriting event handling, it's a good time to fix it.

**Review by avaer — COMMENTED** [review]
Overall looks good.

Should port the mega map screen to the new open tracking system, since it is designed to be modal and exclusive like the other tabs.

**Comment by ohmed** [comment]
@alisaad673 fixed all issues listed above.
can be retested [character btn position is the same, looks better like it is as for me].

**Comment by ohmed** [comment]
@alisaad673 fixed all issues listed above.
Only don't know about warnings [first item], I don't get any or don't know the reproduce steps.

**Comment by ohmed** [comment]
Also fixed:
1. Enter key issue when pressed before onto 'scene-select' panel btns.
2. 'World' panel btn now visible when 'World' panel is opened.
3. Fixed handling 'm' & 'z' btns in chat.

**Comment by avaer** [comment]
Looks good but has conflicts.

**Comment by avaer** [comment]
I fixed the conflicts.

However, this PR regresses the UI animations further than previous PRs: https://github.com/webaverse/app/issues/2592

**Comment by avaer** [comment]
Even after this supposedly passed tests I'm seeing regressions in the animations. It might need another redo, with a strong emphasis on not regressing.

This PR included many other fixes for previously introduced UI regressions but this PR isn't likely to be merged anymore. Can we break those out?

**Comment by ohmed** [comment]
Updated panels open/close animations [+character emotions bar animation].

**Comment by avaer** [comment]
Mouse wheel seems completely broken in this branch. It does not scroll the camera at all.

**Comment by alisaad673** [comment]
Just tested it again. Mouse wheel is working fine now.

**Comment by avaer** [comment]
This PR is still busted in mouse wheel:

```
Unable to preventDefault inside passive event listener invocation.
```

image

**Comment by avaer** [comment]
There are also conflicts.

**Comment by avaer** [comment]
Bugs are fixed after merge.

**Pull request merged**